react-components/src/components/RuleBasedOutputs/utils.ts

```diff
@@ -45,8 +45,8 @@ const checkStringExpressionStatement = (
   );
   const assetTrigger =
     trigger?.type === 'metadata' &&
-    currentTriggerData?.type === 'metadata' &&
-    currentTriggerData?.asset !== undefined
+      currentTriggerData?.type === 'metadata' &&
+      currentTriggerData?.asset !== undefined
       ? currentTriggerData?.asset[trigger.type]?.[trigger.key]
```

Comment:
I know I probably should have mentioned this in an earlier review, but could we put these expressions into a variable with an explanatory name? :sweat_smile:

Reply:
ops. Done:
https://github.com/cognitedata/reveal/pull/4528/commits/9323bc3a1a308c717e935c0d0dfc044ac830ee25